virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc

```diff
@@ -0,0 +1,23 @@
+[id="virt-configuring-vgpu-passthrough"]
+= Configuring vGPU passthrough
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-vgpu-passthrough
+
+toc::[]
+
+Your virtual machines can access a virtual GPU (vGPU) hardware. Assigning a vGPU to your virtual machine allows you to:
+
+* Access a fraction of the underlying hardware's GPU to achieve high performance benefits in your virtual machine.
+
+* Streamline resource-intensive I/O operations.
+
+[IMPORTANT]
+====
+vGPU passthrough can only be assigned to devices that are connected to clusters running in a bare metal environment.
+====
+
+include::modules/virt-assign-vgpu-passthrough-to-vm.adoc[leveloffset=+1]
+
+= Additional Resources
```

Comment:
Lowercase "resources", and this should be `==`; there can only be 1 `=` per assembly.

Also needs a unique section ID, e.g.:

```
[role="_additional-resources"]
[id="additional-resources_virt-configuring-vgpu-passthrough"]
= Additional resources
```

The `role` thing is new too. Both mentioned in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections.

Reply:
Done, thanks.